packages/cf-builder-card/src/CardBuilder.js

```diff
@@ -48,21 +66,7 @@ class CardBuilder extends React.Component {
   }
 }
 
-CardBuilder.propTypes = {
```

Comment:
Option 1 might _look_ good, but it is still not good. It's much better for Flow to know definitively what properties are allowed so that variance works as expected.